libs/protocol-client/src/lib/utils/ipfs.ts

```diff
@@ -0,0 +1,73 @@
+import { Buffer } from 'buffer';
+import { create, IPFSHTTPClient, CID } from 'ipfs-http-client';
+import { ContributionV1SchemaCid } from './constants';
+
+export type MintedContributionSchemaV1 = {
+  schema: `ipfs://Qmc3fKseQ1JyiJiPsmazx23A6twy6YCkTfqDfHUKZDL5xq`;
```

Comment:
`schema` is omitted for all usages, and sometimes replaced like here.

Reply:
This is a type definition here. It is saying the type of the schema is that string. The reason we omit schema sometimes is we only need to define it at the lowest call level which would be at the line below.

https://github.com/Govrn-HQ/govrn-monorepo/pull/273/files#diff-44778fada2dbebb5a7858bcb7ae0fdf3a7e3d05e69bd44e8b4f3fdb70fd6ec93R32